From 63901da067e069e298595618e01c4758c7896ff5 Mon Sep 17 00:00:00 2001
From: thomasanderson <thomasanderson@google.com>
Date: Fri, 28 Apr 2017 11:56:12 -0700
Subject: [PATCH] Fix kernel version condition for including dma-buf.h

Kernel 4.11 merges the commit that added linux/dma-buf.h
https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=ef96152e6a36e0510387cb174178b7982c1ae879

This CL increases the required kernel to include this file to 4.11.

BUG=707604
R=danakj@chromium.org

Review-Url: https://codereview.chromium.org/2851803002
Cr-Commit-Position: refs/heads/master@{#468078}
---
 ui/gfx/linux/client_native_pixmap_dmabuf.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ui/gfx/linux/client_native_pixmap_dmabuf.cc b/ui/gfx/linux/client_native_pixmap_dmabuf.cc
index 31ff4f4395b6..4927daf3a61d 100644
--- a/ui/gfx/linux/client_native_pixmap_dmabuf.cc
+++ b/ui/gfx/linux/client_native_pixmap_dmabuf.cc
@@ -19,7 +19,7 @@
 #include "base/strings/stringprintf.h"
 #include "base/trace_event/trace_event.h"
 
-#if LINUX_VERSION_CODE >= KERNEL_VERSION(4, 6, 0)
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(4, 11, 0)
 #include <linux/dma-buf.h>
 #else
 #include <linux/types.h>
-- 
2.13.0

